docs/index.md

```diff
@@ -87,6 +87,7 @@ variable, use `fix_code`:
     - item
     - item
   ```
+- Quote jobs.*.containers.volumes entries in GitHub Actions workflow yamls to not confuse GitHubs Yaml parser.
```

Comment:
I wouldn't say this is a feature, but a fix for an unexpected behavior, isn't it? If you agree I'd remove this line